latest/index.bs

```diff
@@ -331,24 +317,917 @@ Conforming readers:
 - MAY ignore other groups or arrays under the root of the hierarchy.
 
 
+"coordinateSystems" metadata {#coord-sys-md}
+--------------------------
+
+A "coordinate system" is a collection of "axes" / dimensions with a name. Every coordinate system:
+- MUST contain the field "name", that gives the space name. The values MUST be non-empty and unique.
+- MUST contain the field "axes", whose value is a valid set of "axes" (see below).
+
+<div class=example>
+
+```json
+{
+    "name" : "volume_micrometers",
+    "axes" : [
+        {"name": "x", "type": "space", "unit": "micrometer"},
+        {"name": "y", "type": "space", "unit": "micrometer"},
+        {"name": "z", "type": "space", "unit": "micrometer"}
+    ]
+}
+```
+</div>
+
+The order of the `"axes"` list matters and defines the index of each array dimension and coordinates for points in that
+coordinate system. For the above example, the `"x"` dimension is the first dimension. The "dimensionality" of a coordinate system
```

Comment:
"rank" is probably a better term than "dimensionality"

Reply:
any particular reason? I used "dimensionality" thinking of vector spaces. In math I guess it's the "dimension" of a vector space, but for purposes of clear prose, I went with dimensionality.

[cf seems to use "dimensionality" sometimes (e.g. Table 7.1)](http://cfconventions.org/Data/cf-conventions/cf-conventions-1.8/cf-conventions.html#geometries)

Reply:
yeah, upon reflection I decided that dimensionality is fine :)